src/main.cpp

```diff
@@ -755,18 +738,24 @@ int main(int argc, char *argv[]) {
 
 
   cout << dev_mgr.get_avail_dev_info(deviceIndex).name.c_str() << endl;
-  cout << "OpenCL version: " << dev_mgr.get_avail_dev_info(deviceIndex).ocl_version.c_str() << endl;
-  cout << "Memory limit: " << dev_mgr.get_avail_dev_info(deviceIndex).max_mem << endl;
-  cout << "WG limit: " << dev_mgr.get_avail_dev_info(deviceIndex).wg_size << endl << endl;
+  if (benchmark_mode == false) {
+      cout << "OpenCL version: " << dev_mgr.get_avail_dev_info(deviceIndex).ocl_version.c_str() << endl;
+      cout << "Memory limit: " << dev_mgr.get_avail_dev_info(deviceIndex).max_mem << endl;
+      cout << "WG limit: " << dev_mgr.get_avail_dev_info(deviceIndex).wg_size << endl << endl;
+  }
```

Comment:
Why is this info not printed in benchmark mode?

Reply:
As one might want to log the console output during a benchmark run, it could lead to unexpected disk startups when the buffer is flushed to disk. This causes strange power spikes. As this output is primarily useful for debugging, I disabled the output in benchmark mode. Do you think it is necessary to output this information in benchmark mode?